src/vs/workbench/parts/html/browser/webview-pre.js

```diff
@@ -1,271 +1,282 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
I'm assuming this converting tab spacing from spaces to tabs? can you confirm it's not the other way around.